### PR TITLE
Issue #5023

### PR DESCRIPTION
**Pull request by mycarrysun**
Created failing test RunCest::runJsonReport
Assert that json_decode does not return null from report.json

### CODE REVIEW

File: tests/cli/RunCest.php [hunk 1]
```diff
@@ -292,7 +293,7 @@ public function runTestWithSubSteps(\CliGuy $I, $scenario)
             $scenario->skip("Xdebug not loaded");
         }
 
-        $file = "codeception".DIRECTORY_SEPARATOR."c3";
+        $file = "codeception" . DIRECTORY_SEPARATOR . "c3";
```
Comment (mycarrysun) [inline tests/cli/RunCest.php]: Formatting here was from PSR2 auto format

**Comment by mycarrysun** [comment]
This issue is fixed by a couple PR's in phpunit-wrapper
[#34](https://github.com/Codeception/phpunit-wrapper/pull/34) and [#35](https://github.com/Codeception/phpunit-wrapper/pull/35)

Can anyone merge this? Small changes that fix the JSON report...

**Comment by DavertMik** [comment]
Now those tests should pass )

**Comment by DavertMik** [comment]
Yeah, it is!

**Pull request merged**